spec/book_spec.rb

```diff
@@ -0,0 +1,27 @@
+require './lib/book.rb'
+
+describe Book do
+  let (:book) {Book.new('managment 101', 'rich')}
```

Comment:
No space after `let`. Also, here you can use `described_class.new(...` rather than `Book.new(...`